### PR TITLE
[BUGFIX] fix how index checked

### PR DESCRIPTION
**Pull request by bekzod**

### CODE REVIEW

File: lib/rsvp/events.js [hunk 1]
```diff
@@ -86,15 +86,14 @@ export default {
...
-
-    callbacks = allCallbacks[eventName];
+    let allCallbacks = callbacksFor(this);
+    let callbacks = allCallbacks[eventName];
 
     if (!callbacks) {
       callbacks = allCallbacks[eventName] = [];
     }
 
-    if (callbacks.indexOf(callback)) {
```
Comment (bekzod) [inline lib/rsvp/events.js]: here index is checked incorrectly

Comment (stefanpenner) [reply lib/rsvp/events.js]: interesting, do we need a test?

**Comment by bekzod** [comment]
I will add test

**Comment by bekzod** [comment]
test added :)

**Pull request merged**